index.js

```diff
@@ -26,20 +33,22 @@ module.exports = bundler => {
 		const spinner = ora(chalk.grey('Fixing og:image link')).start();
 		const start = Date.now();
 
-		const htmlPath = path.join(bundler.options.outDir, 'index.html');
-		const html = fs.readFileSync(htmlPath).toString();
-
-		const ogImageTag = getMetaTag(html, 'og:image');
-		const ogImageContent = getMetaTagContent(ogImageTag);
-
-		const ogUrlTag = getMetaTag(html, 'og:url');
-		const ogUrlContent = getMetaTagContent(ogUrlTag);
-
-		const absoluteOgImageUrl = url.resolve(ogUrlContent, ogImageContent);
-		const ogImageTagAbsoluteUrl = ogImageTag.replace(ogImageContent, absoluteOgImageUrl);
-		const patchedHtml = html.replace(ogImageTag, ogImageTagAbsoluteUrl);
-
-		fs.writeFileSync(htmlPath, patchedHtml);
+		glob.sync(`${bundler.options.outDir}/**/*.html`).forEach(file => {
+			const htmlPath = path.resolve(file);
+			const html = fs.readFileSync(htmlPath).toString();
+			const ogImageTag = getMetaTag(html, 'og:image');
+			const ogUrlTag = getMetaTag(html, 'og:url');
+
+			if (ogImageTag && ogUrlTag) {
+				const ogImageContent = getMetaTagContent(ogImageTag);
+				const ogUrlContent = new URL(getMetaTagContent(ogUrlTag));
+				const absoluteOgImageUrl = url.resolve(ogUrlContent.origin, ogImageContent);
```

Comment:
Relying on `ogUrlContent.origin` here isn't safe because we can't be sure this is the site root.

The previous method of resolving `ogImageContent` relative to `ogUrlContent` will always be safe html files in the root but will break in sub directories.

```js
url.resolve(ogUrlContent, ogImageContent);
```

You could probably fix this by working out where `htmlPath` is relative to `bundler.options.outDir` and using that to fix the absolute path.